frontend/src/metabase/visualizations/visualizations/PivotTable/PivotTableCell.tsx

```diff
@@ -59,6 +60,7 @@ export function Cell({
   hasTopBorder,
   onClick,
   onResize,
+  isScrolling = false,
```

Comment:
`Cell` and `BodyCell` are rather "dumb" UI components therefore I think they don't need to know _why_ the tooltip is disabled. All they need to know is whether the tooltip is disabled. So I'd call this property `disabledTooltip` or `showTooltip` or something like that.